README.md

```diff
@@ -71,3 +71,24 @@ docker run --env-file .env --add-host=host.docker.internal:host-gateway --detach
 
 And you're done! This mini-tutorial doesn't support the leaderboard, though. You also may wish to set up a simple
 docker compose file. [Join the discord](https://valk.sh/discord) to learn more!
+
+
+# Using Docker Compose
+You can grab the file from [here](/docker-compose.yml)
+
+## Env File:
+You can grav the variables from [here](/.env.example)
+
+Make sure you replace `<token>` and both `<db_pass>` with your own bot token and database password for postgres. You do **NOT** need to configure postgres, just set those variables up and it will configure for you.
+
+## Finally, start the docker with:
+```bash
+docker compose up
+```
+If there are any errors, ping valkyrie_pilot on the official discord [here](<https://valk.sh/discord>)
```

Comment:
i don't think you need the angle brackets here?

Reply:
The angle brackets, while not necessary, help *some* with netness

Reply:
fair enough